### PR TITLE
COP-12586: warnings implementation to use values from an API

### PR DESCRIPTION
**Pull request by Shaks**
COP-12586: warnings implementation to use values from an API

### CODE REVIEW

File: client/src/pages/targets/utils/RisksUtil.js [hunk 1]
```diff
@@ -79,11 +79,17 @@ const getWarning = (selector) => {
   return selector?.warning || undefined;
 };
 
+const getPncWarning = (informationSheet) => {
```
Comment (charles474) [inline client/src/pages/targets/utils/RisksUtil.js:82]: Missing test for this.

File: client/src/pages/targets/adapters/row/targetRow.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ const toAirpaxRow = (target, handleRadioButton, menuName, checkedRadioBtnName) =
         {RisksUtil.category(selector)}
       </td>
       <td role="cell" className="hods-table__cell">
-        {MovementUtil.hasWarnings(persons)}
+        {pncWarning}
```
Comment (charles474) [inline client/src/pages/targets/adapters/row/targetRow.js:83]: Call `RisksUtil.pncWarning(informationSheet)` this directly here.

File: client/src/pages/targets/adapters/row/targetRow.js [hunk 3]
```diff
@@ -135,7 +135,7 @@ const toRoRoRow = (target, handleRadioButton, menuName, checkedRadioBtnName) =>
...
```
Comment (charles474) [inline client/src/pages/targets/adapters/row/targetRow.js:138]: Call this `RisksUtil.pncWarning(informationSheet)` directly here.

**Review by charles474 — CHANGES_REQUESTED** [review]

File: client/src/pages/targets/utils/RisksUtil.js (hunk 1 above)
Comment (Shaks) [reply client/src/pages/targets/utils/RisksUtil.js:82]: Added

File: client/src/pages/targets/utils/RisksUtil.test.js [hunk 4]
```diff
@@ -161,4 +161,18 @@ describe('RisksUtil', () => {
...
   );
+
+  it('should return Yes if pnc warning is present', () => {
+    const risks = new RisksMother().withHasPncWarnings(true).build();
+    const { informationSheet } = new AirpaxTargetInfoMother().withRisks(risks).build();
+    const warning = RisksUtil.pncWarning(informationSheet);
+    expect(warning).toEqual('Yes');
+  });
+
+  it('should return No if pnc warning is not present', () => {
```
Comment (charles474) [inline client/src/pages/targets/utils/RisksUtil.test.js:172]: I know the value is always a boolean but always try and cover `undefined` & `null`.

**Review by charles474 — APPROVED** [review]
https://github.com/UKHomeOffice/cop-ui/pull/716/files#r1048699034

File: client/src/pages/targets/utils/RisksUtil.test.js (hunk 4 above)
Comment (Shaks) [reply client/src/pages/targets/utils/RisksUtil.test.js:172]: Take care of it next time

**Pull request merged**